src/components/ResultsCount/ResultsCount.tsx

```diff
@@ -6,7 +6,7 @@ type ResultsCountProps = {
   className?: string;
   currentCount: string;
   id?: string;
-  onChange?: () => void;
+  onChange?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
```

Comment:
Instead of React.ChangeEvent use destructuring in the import `import React, { ChangeEvent } from 'react';` and change this to `event:ChangeEvent<HTMLSelectElement>`